src/modules/class.js

```diff
@@ -0,0 +1,69 @@
+import { addTask, editTasks, removeTask } from './controllers.js';
+import { listPlaceholder, inputForm } from './selectors.js';
+
+class Tasks {
+  constructor() {
+    this.tasks = [];
+  }
+
+  createTask(task) {
+    addTask(task, this.tasks);
+
+    this.render();
+  }
+
+  deleteTask(index) {
+    removeTask(index, this.tasks);
+    this.render();
+  }
+
+  updateTask(index, value) {
+    editTasks(this.tasks, index, value);
+    this.render();
+  }
+
+  render() {
+    listPlaceholder.innerHTML = '';
+    this.tasks = JSON.parse(localStorage.getItem('tasks')) || [];
+    this.tasks.forEach((task) => {
+      const taskItem = document.createElement('li');
+      taskItem.setAttribute('data-index', task.index);
+      taskItem.innerHTML = `
+                      <div>
+                        <input type="checkbox" id="check" name="task_check">
+                        <input type="text" class="task-input" value=${task.description}>
+                      </div>
+                        <i class="fa-solid fa-ellipsis-vertical del-icon"></i>  
+                      `;
+      listPlaceholder.appendChild(taskItem);
+      const delIcon = taskItem.querySelector('.del-icon');
+      delIcon.addEventListener('click', (e) => {
```

Comment:
[Optional Changes]

- [ ] The icon used here is meant for options. Basically, according to the original design, it's meant for when you click it, then it gives the option for deleting a todo with the delete icon. But since you did not achieve the design style, you place just the delete icon instead of these three dots. That will improve the user interaction of your app. 👍🏽